buttons/button_base.h

```diff
@@ -30,27 +31,36 @@ class ButtonBase : public Looper,
         current_modifiers |= button_;
       }
       while (DebouncedRead()) {
-	if (millis() - push_millis_ > 300) {
-	  prop.Event(button_, EVENT_HELD);
-	  while (DebouncedRead()) {
-	    if (millis() - push_millis_ > 2000) {
-	      prop.Event(button_, EVENT_HELD_LONG);
-	      while (DebouncedRead()) YIELD();
-	    }
-	    YIELD();
-	  }
-	}
-	YIELD();
-      }
-      while (DebouncedRead()) YIELD();
-      prop.Event(button_, EVENT_RELEASED);
-      if (current_modifiers & button_) {
-        current_modifiers &=~ button_;
-        if (millis() - push_millis_ < 500) {
-          prop.Event(button_, EVENT_CLICK_SHORT);
-        } else {
-          prop.Event(button_, EVENT_CLICK_LONG);
+          if (!short_click){
+            if (millis() - push_millis_ > 300) {
+                prop.Event(button_, EVENT_HELD);
```

Comment:
indentation should be two spaces per level, not four

Reply:
fixed.